pjlib/include/pj/ioqueue.h

```diff
@@ -589,6 +589,19 @@ PJ_DECL(pj_status_t) pj_ioqueue_post_completion( pj_ioqueue_key_t *key,
                                                  pj_ssize_t bytes_status );
 
 
+/**
+ * Clear ioqueue key states. This function will cancel any outstanding
+ * operations on that key, without invoking any completion callback.
+ * After calling this function, application should reinit its all operation
+ * keys, i.e: using pj_ioqueue_op_key_init(), before reusing them.
```

Comment:
So this means that the ioq_op_key_init() needs to be moved from transport_attach() to _start(), right?

Reply:
ah right, missed that.